src/SourceBuild/Arcade/eng/common/templates/job/source-build-run-tarball-build.yml

```diff
@@ -116,6 +116,9 @@ jobs:
   - job: Rebuild_Tarball
     displayName: Rebuild Tarball With Previous
     dependsOn: Build_Tarball
+    # Always attempt to run the bootstrap leg (e.g. even when stage 1 tests fail) in order to get a complete accessment of the build status.
+    # The bootstrap build will shortcut if the stage 1 build failed.
+    condition: succeededOrFailed()
```

Comment:
This isn't clear to me, how will we get a complete assessment of the build status if the bootstrap leg will shortcut when stage 1 fails? Is the goal to have the leg run and fail instantly instead of skipping it?